.github/workflows/ci.yml

```diff
@@ -69,21 +69,15 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, windows-latest, macos-latest ]
-        java: [ 11, 17 ]
+        java: [ '17.0.7', '20.0.1' ]
```

Comment:
do we have to put the minor versions ? Can't we just use `17` and `20` ?

Reply:
`17` and `20` works as well, I've updated it